src/mcts/stoppers/legacy.cc

```diff
@@ -143,10 +142,10 @@ std::unique_ptr<SearchStopper> LegacyTimeManager::GetStopper(
   constexpr int kSmartPruningToleranceMs = 200;
   if (slowmover_ < 1.0 ||
       this_move_time * slowmover_ > kSmartPruningToleranceMs) {
-    this_move_time *= slowmover_;
     // If time is planned to be overused because of slowmover, remove excess
     // of that time from spared time.
     time_spared_ms_ -= this_move_time * (slowmover_ - 1);
```

Comment:
If `time_spared_ms_` becomes negative, this is then added to total_moves_time.
Or is this prevented somewhere?

Reply:
I think it is safe, but I see a way to improve it.

Reply:
And if `slowmover_` is < 1.0, it will add time, is that intended?

Reply:
This is how I understood the intention of the code: To adjust `time_spared_ms_` according to what `slowmover` saves/uses for the next move. Whether this is the best way is an open question.